style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+}
+
+/* NAVIGATION BAR */
+
+.navbar {
+  margin-top: 25px;
+}
+
+.navbar ul {
+  display: flex;
+  list-style: none;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+.my-logo {
+  width: 50px;
+  margin-left: 24px;
+}
+
+.hamburguer-menu {
+  color: #6070ff;
+  display: inline-block;
+  margin-right: 24px;
+  width: 16px;
+}
+
+/* INTRODUCTION SECTION */
+
+#introduction-section {
+  background: url("Images_Background/header-shapes\ mobile.png") no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] Please make sure that the content of the section is aligned with flexbox. This gives us a lot of options to align all our elements, plus, it's responsive.💪 [Check out this link](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) for more information ℹ
- [ ] Kindly make sure that the background image is not cropped. Right now, I can see that the background image is not according to what is given in the design guidelines. See the images below 👇 _Hint: This is related to height_
- [ ] Please consider adding a `border-bottom-left-radius` for the first section. Right now, it is missing. It may not be visible now, but when you will start working on the next milestone of the portfolio project, it will be visible. See the image below for more clarity 💡
- [OPTIONAL] I think it would be much better to use `.svg` instead of `.png` 💪 While `pngs` are capable of handling very high resolutions, they're not infinitely expandable. On the other hand, `svg` files can expand to any size without losing their resolution ℹ

Your design | Design guidelines
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/234743022-87e0f981-9824-442f-888f-4c3a8fccb4b0.png) | ![image](https://user-images.githubusercontent.com/106119105/234743272-21aa6b2e-9098-428f-baee-151ab844d362.png)